libs/designsystem/header-actions/src/header-actions.component.spec.ts

```diff
@@ -0,0 +1,22 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
Tests I have considered: 
1. Set the visibleButtons input to n, and verify that only n buttons are shown in the visible layer
2. Verify that if visibleButtons='n' and n+1 buttons are slotted, a dropdown/button with more-icon appears
3. If max-visible buttons are not supplied, all buttons should be shown and no dropdown should be present (this constraint changes a bit when dynamic behavior is introduced)

Reply:
We need to decide if testing should be postponed, if it is I'll delete this file.